spec/requests/api/v1/search_results_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::SearchResultsController, type: :request do
+  describe 'GET#index' do
+    context 'given a logged in user' do
+      context 'given the user has keywords' do
+        it 'retuns success status' do
+          user = Fabricate(:user)
+          ads_top_urls = ['https://www.thetopvpn.com', 'https://www.nordvpn.com', 'https://www.vnexpress.net']
+          Fabricate(:keyword, ads_top_urls: ads_top_urls, user: user)
+
+          params = { filter: { adwords_url_contains: 'VPN' }, page: 1, per_page: 1 }
+          get api_v1_search_results_path, params: params, headers: create_token_header(user)
+
+          expect(response).to have_http_status(:success)
+        end
+
+        context 'given there are adword urls contain the keyword VPN' do
+          it 'retuns 2 adword urls match the word' do
```

Comment:
Does it actually return the 2 adword urls? 💭 

```suggestion
          it 'returns 2 keywords contains the matched urls' do
```